src/components/MultiInputs/index.tsx

```diff
@@ -17,11 +17,13 @@ const MultiInputs = ({ items, onValuesChanged }: MultiInputProps): JSX.Element =
   const [selectedValues, setSelectedValues] = useState<AnswerRequest[]>([]);
 
   const handleValuesChanged = (answer: Answer, content: string) => {
-    const newSelectedValues = selectedValues;
-    const itemIndex = newSelectedValues.findIndex((value) => value.id === answer.id);
-    if (itemIndex !== -1) {
-      newSelectedValues[itemIndex].answer = content;
-    }
+    const newSelectedValues = [
+      ...selectedValues.filter((value) => value.id !== answer.id),
+      {
+        id: answer.id,
+        answer: content,
+      },
+    ];
```

Comment:
I prefer suggest using `push` method instead of spreading
```suggestion
    const newSelectedValues = selectedValues.filter((value) => value.id !== answer.id).push(
      {
        id: answer.id,
        answer: content,
      })
```

Reply:
@nvminhtue can't, `push()` return `number` type 😓

Reply:
🤔 that's odd, they are nothing but the same

Reply:
@nvminhtue I want to get a new list, but adding `push` method to the end will change the return type to `number`
![image](https://github.com/manh-t/react-survey/assets/60863885/81309f8d-5c13-45df-b083-2bbd6f88d4b9)

Reply:
Ah, my bad, it's enough
```suggestion
	// selectedValues is updated itself
    selectedValues.filter((value) => value.id !== answer.id).push(
      {
        id: answer.id,
        answer: content,
      })
```
But I'm fine with both